tests/lib/screenshot-testing/support/app.js

```diff
@@ -38,8 +38,8 @@ var walk = function (dir, pattern, result) {
 };
 
 var isCorePlugin = function (pathToPlugin) {
-    // if the plugin is a .git checkout, it's not part of core
-    var gitDir = path.join(pathToPlugin, '.git');
+    // skip plugins that have special needs in core build
+    var gitDir = path.join(pathToPlugin, 'tests/travis');
```

Comment:
The variable & function name could be changed to be more accurate

Reply:
Also, I think this method is for running the tests locally so modifying it might affect test running locally w/ git checkouts of pro plugins, eg.